extensions/sendbird/v1/actions/updateUser/updateUser.ts

```diff
@@ -0,0 +1,83 @@
+import { z, ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+import { type Action } from '@awell-health/extensions-core'
+import { type settings } from '../../../settings'
+import { Category, validate } from '@awell-health/extensions-core'
+import { SettingsValidationSchema } from '../../../settings'
+import { FieldsValidationSchema, fields } from './config'
+import {
+  SendbirdClient,
+  isSendbirdChatError,
+  sendbirdChatErrorToActivityEvent,
+} from '../../client'
+
+export const updateUser: Action<typeof fields, typeof settings> = {
+  key: 'updateUser',
+  title: 'Update user',
+  description: 'Updates user using Chat API.',
+  category: Category.COMMUNICATION,
+  fields,
+  previewable: false,
+  onActivityCreated: async (payload, onComplete, onError) => {
+    try {
+      const {
+        settings: { applicationId, chatApiToken, deskApiToken },
+        fields: { userId, profileUrl, nickname, issueAccessToken },
+      } = validate({
+        schema: z.object({
+          settings: SettingsValidationSchema,
+          fields: FieldsValidationSchema,
+        }),
+        payload,
+      })
+
+      const client = new SendbirdClient({
+        applicationId,
+        chatApiToken,
+        deskApiToken,
+      })
+
+      await client.chatApi.updateUser({
+        user_id: userId,
+        nickname,
+        issue_access_token: issueAccessToken,
+        profile_url: profileUrl,
```

Comment:
We need to make sure that when `profileUrl` is an empty string we pass `undefined` and not an empty string.

Reply:
Yes, that's what `makeStringOptional` in fields validation does. It returns `undefined` for empty strings so that the field is not overridden. I used it for all string fields in all actions in this extension